manifest: Update sdk-zephyr revision (automatic upmerge)

NordicBuilder opened the pull request:
Automatic upmerge action

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/nrfconnect/sdk-nrf?pullRequest=10187) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/nrfconnect/sdk-nrf?pullRequest=10187) before we can accept your contribution.<br/><hr/>**Nordic Builder** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/nrfconnect/sdk-nrf?pullRequest=10187) it.</sub>

NordicBuilder commented:
## Test specification

### [CI/Jenkins/NRF](https://jenkins-ncs.nordicsemi.no/job/latest/job/sdk-nrf/job/PR-10187/1/)
- Integration Platforms

### [CI/Jenkins/integration](https://jenkins-ncs.nordicsemi.no/job/latest/job/sdk-nrf/job/PR-10187/1/)
| Test Module | File based changes | Manually selected | West overwrite |
| ----------- | :----------------: | :---------------: | :------------: |
| desktop52_verification | X | X | |
| test-ci-nrfconnect-boot-fw-update | | X | |
| test-fw-nrfconnect-apps | | X | |
| test-fw-nrfconnect-ble | X | X | |
| test-fw-nrfconnect-ble_mesh | X | X | |
| test-fw-nrfconnect-ble_samples | X | X | |
| test-fw-nrfconnect-chip | X | X | |
| test-fw-nrfconnect-fem | | X | |
| test-fw-nrfconnect-nfc | X | X | |
| test-fw-nrfconnect-nrf-iot_cloud | X | X | |
| test-fw-nrfconnect-nrf-iot_libmodem | | X | |
| test-fw-nrfconnect-nrf-iot_lwm2m | X | X | |
| test-fw-nrfconnect-nrf-iot_mosh | X | X | |
| test-fw-nrfconnect-nrf-iot_positioning | X | X | |
| test-fw-nrfconnect-nrf-iot_samples | X | X | |
| test-fw-nrfconnect-nrf-iot_serial_lte_modem | | X | |
| test-fw-nrfconnect-nrf-iot_thingy91 | X | X | |
| test-fw-nrfconnect-nrf-iot_zephyr_lwm2m | X | X | |
| test-fw-nrfconnect-nrf_crypto | X | X | |
| test-fw-nrfconnect-proprietary_esb | | X | |
| test-fw-nrfconnect-rpc | X | X | |
| test-fw-nrfconnect-rs | X | X | |
| test-fw-nrfconnect-tfm | X | X | |
| test-fw-nrfconnect-thread | X | X | |
| test-fw-nrfconnect-zigbee | | X | |
| test-sdk-audio | | X | |
| test-sdk-find-my | X | X | |
| test-sdk-homekit | X | X | |
| test-sdk-wifi | | X | |
| test-sdk-wifi-radio | | X | |
| test-sdk-wifi-scan | | X | |
| test-sdk-wifi-sta | | X | |

**All integration tests**: added because 'CI-all-test' was added manually

[Detailed information of selected test modules](https://jenkins-ncs.nordicsemi.no/job/latest/job/sdk-nrf/job/PR-10187/1/artifact/test-spec/test-spec-summary.yml/*view*/)

*Note: This message is automatically posted and updated by the CI*
